warehouse/models/staging/payments/littlepay/stg_littlepay__customer_funding_source.sql

```diff
@@ -1,8 +1,8 @@
 WITH source AS (
-    SELECT * FROM {{ littlepay_source('external_littlepay', 'customer_funding_source') }}
+    SELECT * FROM {{ source('external_littlepay', 'customer_funding_source') }}
 ),
 
-stg_littlepay__customer_funding_source AS (
+clean_columns_and_dedupe_files AS (
```

Comment:
In my PR #2994 I changed the corresponding CTE names to just `clean_columns` because we are no longer deduping files